src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7526,6 +7526,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_AddressOfInAsync_Title" xml:space="preserve">
     <value>The '&amp;' operator should not be used on parameters or local variables in async methods.</value>
   </data>
+  <data name="WRN_ByValArraySizeConstRequired" xml:space="preserve">
+    <value>Attribute parameter 'SizeConst' must be specified.</value>
```

Comment:
>parameter [](http://example.com/codeflow?start=21&length=9)

SizeConst is a field, not a parameter. Right?

Reply:
It's a field, but I wanted to have this message match the message for when `SizeConst` is not specified in the `ByValTStr` case (which uses the message in `ERR_AttributeParameterRequired1`).

Reply:
> It's a field, but I wanted to have this message match the message for when SizeConst is not specified in the ByValTStr case (which uses the message in ERR_AttributeParameterRequired1).

Fair enough.